horovod/spark/lightning/remote.py

```diff
@@ -74,10 +76,10 @@ def RemoteTrainer(estimator, metadata, ckpt_bytes, run_id, dataset_idx, train_ro
     if sample_weight_col:
         schema_fields.append(sample_weight_col)
 
-    dataloader_cls = _create_dataloader(feature_columns, input_shapes, metadata)
+    data_loader_cls = _create_dataloader(feature_columns, input_shapes, metadata, data_loader_cls)
     make_petastorm_reader = _make_petastorm_reader_fn(transformation, schema_fields,
                                                       batch_size, calculate_shuffle_buffer_size,
-                                                      dataloader_cls)
+                                                      data_loader_cls, loader_num_epochs)
```

Comment:
I did a quick search here, the `make_petastorm_reader` is not used anymore?
This function actually set dataloader as attribute of lightning model.
If I understand correctly, we should rename this function.